config/config/src/test/java/io/helidon/config/ConfigTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This should be `2017, 2019` instead of `2019`.

Reply:
I'm not sure why this file was even changed as part of our PR. I can only assume that someone couldn't get checkstyle to pass because the only changes made were to correct checkstyle errors; although I didn't think checkstyle ran on test classes